__init__.py

```diff
@@ -148,37 +119,26 @@
 from scm.plams.tools.kftools import KFFile, KFHistory, KFReader
 from scm.plams.tools.periodic_table import PT, PeriodicTable
 from scm.plams.tools.plot import (
+    get_correlation_xy,
     plot_band_structure,
-    plot_molecule,
     plot_correlation,
-    get_correlation_xy,
+    plot_molecule,
     plot_msd,
     plot_work_function,
 )
-from scm.plams.tools.reaction_energies import (
-    balance_equation,
-    balance_equation_new,
-    get_stoichiometry,
-    reaction_energy,
-)
+from scm.plams.tools.reaction import ReactionEquation
+from scm.plams.tools.reaction_energies import balance_equation, balance_equation_new, get_stoichiometry, reaction_energy
 from scm.plams.tools.units import Units
 from scm.plams.trajectories.dcdfile import DCDTrajectoryFile
-from scm.plams.trajectories.rkffile import (
-    RKFTrajectoryFile,
-    write_general_section,
-    write_molecule_section,
-)
-from scm.plams.trajectories.rkfhistoryfile import (
-    RKFHistoryFile,
-    molecules_to_rkf,
-    rkf_filter_regions,
-)
+from scm.plams.trajectories.rkffile import RKFTrajectoryFile, write_general_section, write_molecule_section
+from scm.plams.trajectories.rkfhistoryfile import RKFHistoryFile, molecules_to_rkf, rkf_filter_regions
 from scm.plams.trajectories.sdffile import SDFTrajectoryFile, create_sdf_string
 from scm.plams.trajectories.sdfhistoryfile import SDFHistoryFile
 from scm.plams.trajectories.trajectory import Trajectory
 from scm.plams.trajectories.trajectoryfile import TrajectoryFile
 from scm.plams.trajectories.xyzfile import XYZTrajectoryFile, create_xyz_string
 from scm.plams.trajectories.xyzhistoryfile import XYZHistoryFile
+from scm.plams.version import __version__
```

Comment:
Was this automatically reordered? (Makes a bigger diff but looks ok)

Reply:
Yes, apparently I had configured to use isort, which is probably good practice, but now I don't exactly know how to undo it.